packages/gatsby-theme-docs/src/components/limit.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ContentNotifications } from '@commercetools-docs/ui-kit';
+import { useLimitByName } from '../hooks/use-limits';
+
+const Limit = (props) => {
```

Comment:
I'm struggling a bit with the naming here. What does limit imply? Can we find a more specific name?

Reply:
The name is an authoring thing, please speak with @nkuehn about that.